pipeline/dbt/models/intermediate/sources/soliguide/_soliguide__models.yml

```
@@ -54,3 +53,21 @@ models:
           - relationships:
               to: ref('int_soliguide__adresses')
               field: id
+
+unit_tests:
+- name: test_address_without_postal_code_and_city
+  model: int_soliguide__adresses
+  given:
+    - input: ref('stg_soliguide__lieux')
+      rows:
+        - {position__address: '22 rue Sainte-Marthe, 75010 Paris'}
+        - {position__address: '3 Rpe des Mobiles, 16300 Barbezieux-Saint-Hilaire'}
+        - {position__address: ',,, 49610 Mozé-sur-Louet'}
+        - {position__address: '36 Rte de Toulon,'}
+  expect:
+    rows:
+      - {adresse: '22 rue Sainte-Marthe'}
+      - {adresse: '3 Rpe des Mobiles'}
+      - {adresse: ',,'}
+      - {adresse: '36 Rte de Toulon,'}
```

gé-nial :heart_eyes:

On ne veut pas ça plutot : 

```suggestion
      - {adresse: null}
      - {adresse: '36 Rte de Toulon'}
```

?

---

Je peux le mettre, ça donnera des choses plus réalistes :+1: